docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,325 @@
+   ```
+
+   This tutorial passes three flags to the build process to configure the build:
+
+   - `VIAMCPPSDK_BUILD_TFLITE_EXAMPLE_MODULE` to request building the example module for this tutorial.
+   - [`VIAMCPPSDK_USE_DYNAMIC_PROTOS`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md#viamcppsdk_use_dynamic_protos) to request that proto generation happen along with the build.
+   - [`CMAKE_INSTALL_PREFIX`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md#cmake_install_prefix) to install to <file>~/example_workspace/opt</file> instead of the default <file>./install</file> location.
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+Follow the [Viam C++ SDK build instructions](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md) to build the SDK on your Linux system.
+
+While your specific build steps may differ slightly, your installation should generally resemble the following:
+
+1. Clone the Viam C++ SDK to your Linux system:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   git clone git@github.com:viamrobotics/viam-cpp-sdk.git
+   ```
+
+1. Build and run the `bullseye` development Docker container included with the SDK:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cd viam-cpp-sdk/
+   docker build -t cpp . -f etc/docker/Dockerfile.debian.bullseye
+   docker run --rm -it -v "$PWD":/usr/src/viam-cpp-sdk -w /usr/src/viam-cpp-sdk cpp /bin/bash
+   ```
+
+   Alternatively, you can skip running the docker container if you would prefer to use your own development environment.
+
+1. Install all listed dependencies to support the Viam C++ SDK:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   sudo apt-get install git cmake build-essential libabsl-dev libboost-all-dev libgrpc++-dev libprotobuf-dev pkg-config ninja-build protobuf-compiler-grpc
+   ```
+
+1. If you are not using the `bullseye` container included with the SDK, you may need to install a newer version of `cmake` to build the SDK.
+   Run the following to determine the version of `cmake` installed on your system:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cmake --version
+   ```
+
+   If the version returned is `3.25` or later, skip to the next step.
+   Otherwise, run the following commands to add the `bullseye-backports` repository and install the version of `cmake` provided there:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   sudo apt-get install software-properties-common
+   sudo apt-add-repository 'deb http://deb.debian.org/debian bullseye-backports main'
+   sudo apt-get update
+   sudo apt-get install -t bullseye-backports cmake
+   ```
+
+1. Create an <file>opt</file> directory to install the build artifacts to:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   mkdir -p ~/example_workspace/opt/
+   ```
+
+1. Create a <file>build</file> directory to house the build:
```

Comment:
Should people put this build directory into their home directory or into the /example_workspace/opt/ directory or where?

Reply:
Oooh just looked at the Mac instructions and it appears it should go in the sdk folder? So if I followed your instructions literally and never cd'ed anywhere that's where it would go (yay!) but I think being explicit about that would be helpful, since you did just have them create a different directory, and we're not actually supposed to cd into that one.